### PR TITLE
improve text about Community Calls for Comment etc.

### PR DESCRIPTION
**Pull request by stpeter**

### CODE REVIEW

**Comment by stpeter** [comment]
This addresses #139 (not 119, that was a typo on check-in).

File: draft-iab-rfcefdp-rfced-model.md [hunk 1]
```diff
@@ -385,7 +385,11 @@ related to the RFC Series:
    same way a working group of the IETF or research group of the IRTF
    would (see {{RFC2418}}).
 
-3. The RSWG shall then further discuss and develop the proposal. 
+3. The RSWG shall then further discuss and develop the proposal. All
+   participants, but especially RSAB members, should pay special
+   attention to any aspects of the proposal that have the potential
+   to significantly modify policies of long standing or historical
+   characteristics of the Series as described under {properties}.
```
Comment (martinthomson) [inline draft-iab-rfcefdp-rfced-model.md:392]:
```suggestion
   characteristics of the Series as described under {{properties}}.
```

This link will be broken until the parallel PR (#140) merges.

**Review by martinthomson — APPROVED** [review]

**Comment by martinthomson** [comment]
@stpeter, there is a section on the right in the GitHub UI ( Linked issues ) that you can use to link this to the correct issue (#139) so that it will close that issue when this merges.

**Comment by becarpenter** [comment]
Looks good to me

File: draft-iab-rfcefdp-rfced-model.md [hunk 2]
```diff
@@ -472,27 +476,51 @@ if issues are identified during RSAB review.
...
+A comment period will last not less than two weeks and should be
+longer if wide outreach is required. Comments will be publicly
 archived on the RFC Editor website.
 
+The RSAB is responsible for considering comments received during 
+a community call for comment. If RSAB members conclude that such
+comments raise important issues that should be addressed, they 
+should do so by discussing those issues within the RSWG or lodging 
+a position of "CONCERN" during RSAB balloting.
+
```
Comment (elear) [inline draft-iab-rfcefdp-rfced-model.md:523]: This text needs to be in Section 3 under the criteria for a CONCERN.

**Comment by elear** [comment]
A reminder: all comments not of an editorial nature should be brought to the list.

**Pull request merged**